### PR TITLE
Update Gnome runtime to 40

### PR DESCRIPTION
**Pull request by picsel2**

### CODE REVIEW

**Comment by muelli** [comment]
@xjuan or @bilelmoussaoui This here LGTM.
If you want, you can add me to the list of maintainers here, so that I can merge this type of change.

**Comment by albfan** [comment]
Will clear if you explain if you tested and it works as expected.

**Comment by Eonfge** [comment]
bot, build
---
@barthalion @bilelmoussaoui So, give it a quick spin and let's push this out

File: org.gnome.Glade.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "app-id" : "org.gnome.Glade",
     "runtime" : "org.gnome.Platform",
-    "runtime-version" : "3.38",
+    "runtime-version" : "40",
```
Comment (hfiguiere) [inline org.gnome.Glade.json:4]: 41 is out now. Better jump to it I think.

**Comment by picsel2** [comment]
> Will clear if you explain if you tested and it works as expected.

I've created this MR because I was bothered by the old runtime that Glade kept on my system. I have no experience in using Glade and have no interest in using it right now. So I'm in no position to say that I've _tested_ it. The only "testing" that I've done was to check if it starts and to click around a bit.

> 41 is out now. Better jump to it I think.

I think so too. I suggest that someone that is really interested in this should create a new MR for the 41 runtime. Otherwise I could edit this one. What would you prefer @albfan ?

**Comment by albfan** [comment]
I know runtime 40 bundles 3.24 (gtk3) and 40 (gtk4)

Not sure for 41

What I mean is that move to runtime 40 doesn't really change anything (still uses gtk3)

If someone can confirm that's still the case for 41 move to that makes more sense, yes.

**Comment by barthalion** [comment]
Both still ship GTK3 and GTK4. The primary difference is that 40 will be EOL'ed when 42 is released, so while not urgent, it's something that needs to be eventually handled to keep receiving security updates.

**Comment by picsel2** [comment]
I've created a new PR ( #18 )  with 41.

**Comment by albfan** [comment]
superseded by #18

**Pull request closed without merging**